server/game/Card.js

```diff
@@ -755,4 +755,6 @@ class Card extends EffectSource {
     }
 }
 
+Card.HOUSES = ['brobnar', 'dis', 'logos', 'mars', 'sanctum', 'saurian', 'shadows', 'staralliance', 'untamed'];
```

Comment:
This probably doesn't belong here - let's move it to a separate constants file if we don't have one already.  It's ok for it to be split client/server as at some point they will become separate projects

Reply:
Done